test/converters/test_converters.py

```diff
@@ -110,10 +110,10 @@ def test_inequality_binary(self):
                 "x0",
                 "x1",
                 "x2",
-                "x1x2@int_slack",
-                "x0x2@int_slack",
-                "x0x1_x1x2LE@int_slack",
-                "x0x1_x1x2GE@int_slack",
+                "x1x2@integer_slack",
```

Comment:
This is the public `name` of the variables we create during the conversion right. Hopefully no user is accessing these variables and using the names, in which case, like the test here things would fail for them. Since this is really an internal convertor issue - and I guess the names really are just a choice to have something meaningful/readable - it seems fine to me. If anyone is using them, which I think is unlikely, then its easy enough for them to change as was done here.

Reply:
Thank you for your feedback. We don't have to take the risk to change the public name now. I reverted it.